### DISCUSSION TITLE
[Simple Window Switcher] PWAs (Progressive Web Apps) should be treated differently for Alt+`

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [valinet/ExplorerPatcher#1124] [Ideas]
Currently, PWAs made from Chrome apps are treated the samely as other Chrome web browser processes.

I suggest PWAs be excluded when I hit "Alt+`" on the Chrome browser, just like on MacOS.

**Comment by valinet** [comment]
I agree. It should use the same logic for grouping things as the taskbar. Make a PR with the necessary changes and I will merge it. Thanks.
